### PR TITLE
Gs 643

### PR DESCRIPTION
**Pull request by sparkleGoat**
Takada Runbook in the documentation. I did not delete the opsguide yet, I can do that in another ticket, there is still some useful tidbits. I am not sure if all the requirements are present for what they would want. Anymore direction for the tasks is appreciated.

### CODE REVIEW

**Comment by sparkleGoat** [comment]
Changing somethings and will add reviewers when ready

File: docs/source/admin/runbook.rst [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+	* HDFS NameNode
+	* ZooKeeper
+	* PostgreSQL
+
+If a process in say, the Data Storage and Coordination Layer, is stopped before all processes in the Data Services and Application Layers, system state may become unstable or corrupt.
+All processes in one layer should be stopped before stopping any processes in the next layer.
+
+Sometimes a single worker process in a lower layer can be stopped and restarted without stopping higher layers.
+See "Fixing a simple, single-server failure".
+
```
Comment (keithbucher) [inline docs/source/admin/runbook.rst:176]: I like the way you grouped these into layers and integrated that into startup/shutdown.  Makes it much more intuitive for someone unfamiliar with the Hadoop stack.

The only suggestion I would make is that if we are including HDFS HA configurations in this, we might need to separate ZooKeeper and PostgreSQL into their own layer along with HDFS JournalNodes.  I believe that layer would need to start prior to a HA HDFS NameNode.

File: docs/source/admin/buildbootstrapguide.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    sudo yum install -y /home/staging/koverse-server-$KOVERSE_VERSION.rpm
+    sudo yum install -y /home/staging/koverse-webapp-$KOVERSE_VERSION.rpm
+    sudo rm -f /home/staging/*.rpm
+Install Koverse Ambari Stack::
+
+  sudo mkdir /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE
+  sudo cur https://koverse-bdaas.s3.amazonaws.com/hdp3/koverse-ambari-stack-3.0.tar.gz -o /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE/KOVERSE.tar.gz
+  sudo tar zxvf /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE/KOVERSE.tar.gz -C /var/lib/ambari-server/resources/stacks/HDP/3.1/services/KOVERSE
+  sudo rm -f $AMBARI_STACK_DIR/KOVERSE/KOVERSE.tar.gz
+
```
Comment (keithbucher) [inline docs/source/admin/buildbootstrapguide.rst:86]: Does it make sense to include information about setting the ulimit configuration for the koverse and accumulo users here?  I feel like that comes up consistently on systems that undergo significant load.

File: docs/source/admin/runbook.rst [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+The main application logs for the koverse-server are in */var/log/koverse-server/koverse-server.log*.
+
+By default, logging levels are set to "INFO".  If logging levels need to be changed,
+
+#. SSH to the Koverse Server host
+#. vi /opt/koverse-server/conf/log4j.xml
+#. Change the Log4J configuration as needed.
+#. Restart Koverse Server (If you do not restart, the new log level property will not take effect.)
+
+Additional logs may be found in */opt/koverse-server/logs/server.err* and */opt/koverse-server/logs/server.out*. These files are where the stderr and stdout of the Koverse Server process are redirected to and may contain messages, especially in the case of fatal startup issues. Additionally, the server.err file contains the output of any Spark drivers that run and may be useful in debugging issues with Spark Transforms.
```
[the remark this answers is not in the document]
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:33]: I had not reviewed this section, thank you for noting this point. The logs on the PwC/TC system are not in opt at all but in /var/log/koverse-server/ . I checked with Jason and he confirmed this is the case for docker images. I will make a seperate section for docker container installs log directory.

[the remark this answers is not in the document]
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:33]: Chatting with Brenna to see if the Developer Docker Image is the best place for this information.
I have corrected the paragraphs in the admin guide now.

File: docs/source/admin/runbook.rst [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+Ambari Health Info
+^^^^^^^^^^^^^^^^^^^
+
+To monitor the cluster health you can view the service panel. The green light indicates that the service is up and running successfully.
+
+.. image:: /_static/Runbook/image1.png
+
+To view the cluster meterics go to the Dashboard view on the Ambari Webpage. There you can customize and view various useful information like HDFS Utilization, CPU/Network/Memory Usage and Cluster Load.
+
+.. image:: /_static/Runbook/image2.png
```
[the remark this answers is not in the document]
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:76]: Yep!

File: docs/source/admin/runbook.rst [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+Ambari Alerts
+^^^^^^^^^^^^^^
+
+Ambari uses the service indicators to convey the status of a service, when issues arise you can view detailed information on the Alerts page.
+
+.. image:: /_static/Runbook/image3.png
+
+A user can also set up email or slack alerting that is customizable.
+
+.. image:: /_static/Runbook/image4.png
```
[the remark this answers is not in the document]
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:90]: Corrected this

File: docs/source/admin/runbook.rst [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+
+**Accumulo**
+
+Use the `Accumulo Export Tables <http://accumulo.apache.org/1.6/examples/export.html>`_ feature to backup the "kv_*" tables.
+
+**Configuration Files**
+
+Copy the entire koverse-server directory - specifically the /conf directory must be included.
+
+
```
[the remark this answers is not in the document]
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:129]: Corrected!

File: docs/source/admin/runbook.rst (hunk 1 above)
Comment (sparkleGoat) [reply docs/source/admin/runbook.rst:196]: I did not author this :) I just copied it from the opsguide.  I will create a layer for zk, postgres and the journal nodes. There is a ticket for HA, I can keep this in mind for those.

File: docs/source/admin/buildbootstrapguide.rst (hunk 2 above)
Comment (sparkleGoat) [reply docs/source/admin/buildbootstrapguide.rst:86]: Yes it does. Will integrate this

File: docs/source/admin/runbook.rst (hunk 1 above)
Comment (keithbucher) [reply docs/source/admin/runbook.rst:203]: cool, looks good

**Pull request merged**